test/ruby_lsp_rails/code_lens_test.rb

```diff
@@ -26,7 +27,7 @@ class Test < ActiveSupport::TestCase
 
         response = RubyLsp::Executor.new(@store, @message_queue).execute({
           method: "textDocument/codeLens",
-          params: { textDocument: { uri: "file:///fake.rb" }, position: { line: 0, character: 0 } },
+          params: { textDocument: { uri: @uri.to_s }, position: { line: 0, character: 0 } },
```

Comment:
`RubyLsp::Executor#run` already calls `to_s` on the `uri`:

https://github.com/Shopify/ruby-lsp/blob/fafc9e1c629fd245aa6f9a425a2d11783c3a418c/lib/ruby_lsp/executor.rb#L38

Reply:
Yeah, that's correct 👍